db/docs/source/introduction.rst

```diff
@@ -72,7 +80,7 @@ The NZ Building Outlines dataset is being procured and released in stages. Image
 Accuracy Specification
 ---------------------------
 
-The Building Outlines contain spatial detail reflective of the visible characteristics of building outlines as seen from the source imagery to an accuracy of 1 metre. Outlines are captured in full where they are partially occluded by vegetation or require additional viewer interpretation, and squared off at 90 degrees if the corner angles are between 80-100 degrees.
+The Building Outlines contain spatial detail reflective of the visible characteristics of building outlines as seen from the source imagery to an accuracy of 1 metre. Outlines are captured in full where they are partially occluded by vegetation or require additional viewer interpretation, and squared off at 90 degrees if the corner angles are between 80-100 degrees. Effort has been made to adhere to the building outline criteria described above, and future updates may include adjustments of building outlines as imagery resolution allows.
 
 
 Valid Geometry
```

Comment:
the formatting for this looks strange - should it be bullet points?

Reply:
Changing this to mention accuracy between 90 and 110%